### PR TITLE
sof-cml-da7219-max98357a topology is merged only in the cml-007-stable-drop branch. And not pushed to master.

### PR DESCRIPTION
**Pull request by kumarsac**
1. Cherrypicked the topology from cml-007-stable-drop and resolved conflict.
2. Fix the Macro Syntax change for topology as per the commit 77b6db8 ("topology: enable pcm range and pipeline rate and remove frame count")

### CODE REVIEW

File: tools/topology/sof-cml-da7219-max98357a.m4 [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     frames, deadline, priority, core)
+
+# Low Latency playback pipeline 1 on PCM 1 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	1, 1, 2, s32le,
+	48, 1000, 0, 0)
```
Comment (lgirdwood) [inline tools/topology/sof-cml-da7219-max98357a.m4]: This needs updated fro master, PCM_ADD args have changed.

Comment (kumarsac) [reply tools/topology/sof-cml-da7219-max98357a.m4]: This commit i have cherry picked from cml-007-drop-stable branch. (https://github.com/thesofproject/sof/commit/05c8e9690dfb217232613f28d1c48424777f92ea)
Yes this commit does not include the PCM_ADD args change. For this i have added second commit in this pull request to add required arg changes.
(https://github.com/thesofproject/sof/pull/2315/commits/d18097084b53885ec8da049313f8bf59449c16ef)

Comment (lgirdwood) [reply tools/topology/sof-cml-da7219-max98357a.m4]: these should just be squashed otherwise they will break bisect.

Comment (kumarsac) [reply tools/topology/sof-cml-da7219-max98357a.m4]: Agreed. i will squash to single commit.

**Comment by lgirdwood** [comment]
@kumarsac sorry, I confused, what is the difference to #2314 ? They both look like the same topology ?

**Comment by kumarsac** [comment]
> @kumarsac sorry, I confused, what is the difference to #2314 ? They both look like the same topology ?

Yes Both are same changes. This one is for master and #2314 for SOF 008 release branch.
 As cml-008-patch-backport is SOF 008 release branch for CML chrome i have raised pull request for that branch also.
As we need to share the branch and commit details for FW binary release.

**Comment by lgirdwood** [comment]
@kumarsac ok, thanks, just the question on reuse with GLK version as the only open.

**Comment by kumarsac** [comment]
> @kumarsac ok, thanks, just the question on reuse with GLK version as the only open.

This is a released topology in SOF 007 (https://github.com/thesofproject/sof/commit/05c8e9690dfb217232613f28d1c48424777f92ea) which is pushed to google. I preferred the same topology with PCM arg fixes.
@macchian  Please provide your inputs on this...

**Comment by jajanusz** [comment]
@kumarsac The same problems as with #2314 . Also do we need both of them? I'm not sure if you wanted to just align topo on master or you need it also on backport branch.

**Comment by kumarsac** [comment]
> @kumarsac The same problems as with #2314 . Also do we need both of them? I'm not sure if you wanted to just align topo on master or you need it also on backport branch.

This PR can be closed as its been taken care in : https://github.com/thesofproject/sof/pull/2397

**Pull request closed without merging**